### ISSUE TITLE
Teachers should be able to pull in last year's information

### ISSUE THREAD

**Issue by Razpudding** [cmda-the-lab/study-guide-frontend-input#22]
It would be great if teachers could get all the info from last year to be filled in already. This is somewhat tricky because names can change over time. I'll hack something together.

**Comment by Razpudding** [comment]
Some more ideas about how this could work:
- In the simplest form it would do a lookup after you fill in the name of a module
- If the server finds a matching course it'll send back the data to the form
- The form will present a popup saying something like: The course you're filling in looks like a course from a previous year (link to that course in the old static studiegids). Would you like me to prefill some fields based on that information?
- If a user selects yes, the relevant fields will be filled in.
- Possibly the affected fields would be highlighted or the user asked to review them. This might be tricky because of field validation.

**Comment by wooorm** [comment]
This is an issue that can be solved for 2020-2021. Which I feel is far away, so not something we really need to worry about now.

I did set up the new cmd-curriculum-api by academic year. And every module has a SIS ID / Catalog ID. That means that in the future, you’d select a year and choose an initiated module for that year. When you select a module, we know its Catalog ID, and could look up that Catalog ID in previous academic years.  It would be pretty doable to add an “import” feature this way!

Closing, as it’s not actionable currently.
